docs/source/newsfragments/4517.feature.1.rst

```diff
@@ -0,0 +1 @@
+:meth:`~cocotb.handle.GpiGPIDiscovery._get` accepts optional parameter :data:`~cocotb.handle.GpiGPIDiscovery._get.discovery_method` that takes values from :class:`~cocotb.handle.GpiGPIDiscovery` and defaults to ``AUTO``. ``NATIVE`` can be used to forbid cross-language search.
```

Comment:
```suggestion
:meth:`~cocotb.handle.GPIDiscovery._get` accepts optional parameter :data:`~cocotb.handle.GPIDiscovery._get.discovery_method` that takes values from :class:`~cocotb.handle.GPIDiscovery` and defaults to ``AUTO``. ``NATIVE`` can be used to forbid cross-language search.
```

Reply:
oh yeah these are actually typos. good catch

Reply:
but also from the wrong object, made a manual change.